visivo/parsers/yaml_ordered_dict.py

```diff
@@ -0,0 +1,59 @@
+import yaml
+import json
+from collections import OrderedDict
+
+
+class YamlOrderedDict(OrderedDict):
+    """
+    An OrderedDict that was loaded from a yaml file, and is annotated
+    with file/line info for reporting about errors in the source file
```

Comment:
This is great, because I could see us wanting to use this context in other functionality outside of error messages (ie search functions, info logs ect.)